src/views/ProxyPage.vue

```diff
@@ -307,20 +452,43 @@ export default {
             const { lines, errors } = parseDecklist(this.config.decklist);
             this.errors = errors;
 
+            const lookupCards = await ScryfallDatasetAsync();
+
+            console.log(lines);
+
             for (let line of lines) {
-                const cardLookup = (await ScryfallDatasetAsync()).cards[line.name];
+                let cardLookup = lookupCards.cards[line.name];
+                if (!cardLookup && line.name.includes(" // ")) {
+                    cardLookup = lookupCards.cards[line.name.split(" // ")[0]];
+                }
```

Comment:
I think I've got the split cards working properly now with all of their names, is this to catch a specific scenario?

Reply:
Yeah, first of all: This is just quick and dirty, I'm sorry but I don't have a lot of time ;-)
But to answer your question, yes I had a problem with this:
`1 Hostile Hostel // Creeping Inn (prm) 94088`

Your front-back cards in the lookup table are stored with only their front-face name, not the combined one Archidekt is using. Instead of doing it this way you could, of course, also modify the lookup table to store the combined name ("Hostile Hostel" & "Hostile Hostel // Creeping Inn" referencing the same card data)

Reply:
These were some of the scenarios that PR aimed to fix by the way.

```
Start / Finish
Start/Fire () 123

3 bind
2 bind // liberate

1 Erase (Not the Urza's Legacy One)
1 Erase (Not the Urza's Legacy One) (UNH) 10
2 Erase
```

When I tried your branch last night they all still resolved to the correct cards so it does seem to be still differentiating them properly.

Reply:
I ran your tests, they should all still resolve, even if I would like to drop the `Erase (…)` card... it is a real troll, right ;-) 

If you want to make the system really good you could generate a regular expression when generating the cards, this regex should contain all possible set codes.

Reply:
Huh, now that's a idea. Runtime has access to the set list as well if it wanted to do some non-regex sanity checks too. 

I tried removing the special handling for "//" I mentioned and your branch works without it too. 

I'm going to go ahead and merge this in and just clean that up later. There's also a slight regression in the way you select a default set selection (Teferi's Protection had a digital first printing that should be ineligible), so I'll try and correct that too.

Your change reminds me that I really want to get the card list out of `main` and I should probably split up the Set Code and Collector Number to different json properties. I think that would have made this easier.

Reply:
Ah, I was wrong - DFCs weren't working right. I was confusing them with split cards and not actually trying them out properly. Rather than run two lookups on the page I just fixed them in the card list.

I also added a configuration toggle so that matching the pasted version can be opted out of. I prefer oldest printing, so it's nice to have the option to do that automatically still.